File: crates/deltoids/tests/diff_cases/cases/232-typescript-jest-new-test-in-replace/3-updated.ts
describe("client", () => {
  it("recovers via the slow path", () => {
    setupStub();
    expect(result).toEqual({ githubId: 1, login: "ghost" });
  });

  it("raises when recovery returns nothing", () => {
    setupStub();
    expect(() => run()).toThrow(RecoveryError);
  });

  it("exposes the total count", () => {
    expect(client.total).toEqual(1);
  });
});

File: crates/deltoids/tests/diff_cases/cases/230-ruby-rspec-new-test-in-replace/3-updated.rb
RSpec.describe Thing do
  it "recovers via the slow path" do
    setup_stub
    expect(result).to eq(github_id: 1, login: "ghost")
  end

  it "raises when recovery returns nothing" do
    setup_stub
    expect { run }.to raise_error(RecoveryError)
  end

  it "exposes the total count" do
    expect(client.total).to eq(1)
  end
end
